packages/buildessential.rb

```diff
@@ -2,7 +2,6 @@
 
 class Buildessential < Package
   description 'A collection of tools essential to compile and build software.'
-  homepage ''
```

Comment:
Why was this line removed?

Reply:
I might have done that just now while deleting an empty space and failing. Let me fix that.